server/controllers/notification.js

```diff
@@ -0,0 +1,176 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const {
+    user,
+    body: { notifyType, title, description, receivers },
+  } = req;
+
+  const sender = await Profile.findOne({ userId: user.id });
+
+  if (!sender) {
+    res.status(401);
+    throw new Error("Not authorized");
+  }
+
+  const notification = await new Notification({
+    sender: sender.id,
+    notifyType,
+    title,
+    description,
+    receivers,
+  }).save();
+
+  if (notification) {
+    res.status(201).json({
+      success: {
+        notification: {
+          _id: notification.id,
```

Comment:
This might be a typo:
```suggestion
          id: notification._id,
```